samples/JumboBuild/JumboBuild.sharpmake.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Ubisoft. All Rights Reserved.
+// Licensed under the Apache 2.0 License. See LICENSE.md in the project root for license information.
+
+using Sharpmake;
+
+namespace JumboBuild
+{
+    [Sharpmake.Generate]
+    public class JumboBuildProject : Project
+    {
+        public JumboBuildProject()
+        {
+            Name = "JumboBuild";
+
+            AddTargets(new Target(
+                    Platform.win32 | Platform.win64,
```

Comment:
please remove win32. it is useless for this sample purpose. Also, only keep Optimization.Release.
This will slowdown the ci for nothing.